cpp/cmake/thirdparty/get_cuvs.cmake

```diff
@@ -56,13 +65,14 @@ endfunction()
 # To use a different CUVS locally, set the CMake variable
 # CPM_cuvs_SOURCE=/path/to/local/cuvs
 find_and_configure_cuvs(VERSION          ${CUML_MIN_VERSION_cuvs}
-      FORK             rapidsai
-      PINNED_TAG       branch-${CUML_BRANCH_VERSION_cuvs}
+      FORK             benfred
+      PINNED_TAG       static_lib
```

Comment:
are there changes needed in cuVS to support this?

Reply:
yes https://github.com/rapidsai/cuvs/pull/382

Reply:
Now that the cuVS side is merged, just a reminder to revert this once the nightly is deployed

Reply:
pin removed in latest commit